ServerCore/Pages/Teams/Delete.cshtml

```diff
@@ -0,0 +1,64 @@
+﻿@page "{eventId}/Teams/Delete"
+@model ServerCore.Pages.Teams.DeleteModel
+
+@{
+    ViewData["Title"] = "Delete";
+}
+
+<h2>Delete</h2>
+
+<h3>Are you sure you want to delete this?</h3>
+<div>
+    <h4>Team</h4>
+    <hr />
+    <dl class="dl-horizontal">
+        <dt>
+            @Html.DisplayNameFor(model => model.Team.Name)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Team.Name)
+        </dd>
+        <dt>
+            @Html.DisplayNameFor(model => model.Team.Password)
```

Comment:
Do we actually need to display anything other than name on this page?

Reply:
The CRUD wizard generates all of this and I decided to leave it in for now.